test/distributed/test_inductor_collectives.py

```diff
@@ -214,6 +232,126 @@ def compile(func, example_inputs):
             inductor_out = compiled_fn(*inputs)
             self.assertTrue(same(eager_out, inductor_out, tol=0.001))
 
+    @unittest.skipIf(not has_triton(), "Inductor+gpu needs triton and recent GPU arch")
+    @skip_if_lt_x_gpu(2)
+    # TODO: somehow inductor bg compile threads are causing hangs at exit with distributed work dtor
+    @patch.object(torch._inductor.config, "compile_threads", 1)
+    def test_all_to_all_single_inductor(self):
+        def example(inp, split_sizes_tensor, *, tag, ranks, group_size):
+            a2a = torch.ops.c10d_functional.all_to_all_single(
+                inp,
+                output_split_sizes=split_sizes_tensor,
+                input_split_sizes=split_sizes_tensor,
+                tag=tag,
+                ranks=ranks,
+                group_size=group_size,
+            )
+            a2a = torch.ops.c10d_functional.wait_tensor(a2a)
```

Comment:
Note to self: add `out = a2a / a2a.sum(dim=0)` to test persistent reduction after a2a, to see whether Inductor can reason about the symbolic sizevar in output shape